test/stress_test/sanity.ts

```diff
@@ -0,0 +1,136 @@
+import { Config } from '@holochain/tryorama'
+import * as R from 'ramda'
+import { Batch } from '@holochain/tryorama-stress-utils'
+
+
+const trace = R.tap(x => console.log('{T}', x))
+const delay = ms => new Promise(r => setTimeout(r, ms))
+
+module.exports = (scenario, configBatch, N, C, I, testConfig) => {
+    const totalInstances = N*C*I
+    const totalConductors = N*C
+
+    let spinUpDelay = 0
+    let retryDelay = 10000
+    let retries = 3
+    let commitCount = 0
+    if (testConfig.retryDelay != undefined) {
+        testConfig.retryDelay
+    }
+    if (testConfig.retries != undefined) {
+        retries = testConfig.retries
+    }
+    if (testConfig.spinUpDelay != undefined) {
+        spinUpDelay = testConfig.spinUpDelay
+    }
+    if (testConfig.commitCount != undefined) {
+        commitCount = testConfig.commitCount
+    }
+
+    scenario('all hashes are available somewhere gettable', async (s, t) => {
+        const players = R.sortBy(p => parseInt(p.name, 10), R.values(await s.players(configBatch(totalConductors, I), false)))
+
+        // range of random number of milliseconds to wait before startup
+        // const startupSpacing = 10000
+        // number of milliseconds to wait between gets
+        const getWait = 20
+
+        await Promise.all(players.map(async player => {
+            //await delay(Math.random()*startupSpacing)
+            return player.spawn()
+        }))
+        console.log("============================================\nall nodes have started\n============================================")
+        console.log(`beginning dht sanity test`)
+        if (spinUpDelay == 0) {
+            console.log("no spin-up delay given using tryorama consistency waiting")
+            await s.consistency()
+        } else {
+            console.log(`spin up delay ${spinUpDelay}`)
+            await delay(spinUpDelay)
+        }
+
+        if (commitCount > 0) {
+            console.log(`asking all nodes to commit ${commitCount} entries`)
+            let i = 0
+            while (i < commitCount) {
+                i+=1
+                // Make every instance of every conductor commit an entry
+                const commitResults = await batch.mapInstances(instance =>
+                                                               instance.call('main', 'commit_entry', { content: trace(`entry-${instance.player.name}-${instance.id}-${i}`) })
+                                                              )
+                //const hashes = commitResults.map(x => x.Ok)
+            }
+        }
+
+        const batch = new Batch(players).iteration('series')
```

Comment:
relying on variable hoisting to use batch earlier, would be good to move this up higher

Reply:
might try parallel iteration